op-chain-ops/safe/batch_helpers.go

```diff
@@ -141,7 +141,6 @@ func createContractInput(input abi.Argument, inputs []ContractInput) ([]Contract
 		return nil, err
 	}
 
-	// TODO: could probably do better than string comparison?
```

Comment:
I guess a better method is to check `input.Type.T == abi.TupleTy`, like we do [here](https://github.com/ethereum-optimism/optimism/blob/develop/op-chain-ops/safe/batch_helpers.go#L178). Either is fine imo.

Reply:
Going to merge this to get the build passing and then will take a stab at using that better method as you suggest.